Lesson_4/HomeWork_4_6.py

```diff
@@ -0,0 +1,23 @@
+# Task 6. Write a program to convert currencies:
+# UAH --> USD
+# USD --> UAH
+# UAH --> euro
+# euro --> UAH
+
+USD_rate = float(38.50)
+euro_rate = float(42.00)
+UAH_rate = float(1.00)
```

Comment:
When you wrote 38.00 Python understood that this is float. You don't need to write float(38.50). Python has dynamical typization